conmon/conmon.c

```diff
@@ -152,11 +175,11 @@ int main(int argc, char *argv[])
 	/* Create the container */
 	if (terminal) {
 		snprintf(cmd, CMD_SIZE,
-			 "%s create %s --pid-file pidfile --console %s",
-			 runtime_path, cid, slname);
+			 "%s create %s %s --pid-file %s --console %s",
```

Comment:
> But this is all semantics and should be discussed in the runtime-spec and not here.

Agreed. Leaving the semantic interpretation up to the runtime implementers is not helping any of us, so the sooner we clear this the better.